static/home/css/style.css

```diff
@@ -191,28 +213,25 @@ nav{
 }
 .navbar li{
 	display: inline;
-	/*padding: 0 5px;*/
 	color: black;
+	margin: 0px 0px;
 }
 .navbar a{
 	display: inline-block;
-	padding: 6.5px;
+	font-size: 18px;
+	padding: 10.5px;
 	transition: 0.4s;
 }
-.navbar a:hover{
-	background-color: red;
-}
 
 /*button-controlled nav bar*/
 .navbar2{
 	position: absolute;
-	top: 32px;
+	top: 45px;
 	left: 0px;
 	padding: 0;
 	width: 100%;
 	background-color: rgba(255,140,0,0.9);
-	box-sizing: border-box;
-	height:0px;
+	/*box-sizing: border-box;*/
```

Comment:
Don't delete anything which is not created by you. Just put them in the comment. And delete all those which are written by you but later on commented.